CX/Telegram/server.js

```diff
@@ -0,0 +1,103 @@
+/**
+ * TODO(developer): 
+ * Add your service key to the current folder.
+ * Uncomment and fill in these variables.
+ */
+// const projectId = 'my-project';
+// const location = 'global';
```

Comment:
How about rename it to `locationId`? so that it would be more consistent with `projectId` and `agentId`.